components/edit-collective/sections/SendingMoney.js

```diff
@@ -65,7 +65,7 @@ class SendingMoney extends React.Component {
           connectedAccounts={this.props.collective.connectedAccounts}
           services={services}
         />
-        {!services.includes('paypal') && (
+        {services.includes('paypal') && (
```

Comment:
The condition here was a mistake: If PayPal is not connected (i.e. there is no connected account for it) we should not display any message about payouts.